plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/table.tsx

```diff
@@ -71,6 +78,27 @@ export function InstalledSubscriptionsTable( props: {
 		},
 	];
 
+	if ( ! props.isLoading && ( ! props.rows || props.rows.length === 0 ) ) {
+		const marketplaceBrowseURL = getNewPath( {}, MARKETPLACE_PATH, {} );
+		const noInstalledSubscriptionsHTML = createInterpolateElement(
+			__(
+				'No extensions or themes installed. <a>Browse the Marketplace</a>',
+				'woocommerce'
+			),
+			{
+				p: <p />,
+				// eslint-disable-next-line jsx-a11y/anchor-has-content
+				a: <a href={ marketplaceBrowseURL } />,
+			}
+		);
+
+		return (
+			<EmptyTable numberOfRows={ 4 }>
+				{ noInstalledSubscriptionsHTML }
+			</EmptyTable>
+		);
```

Comment:
I haven't made your life easier when structuring the page. I think is a good compromise, even though it causes **Available to use** table to be visible at all times. I know you are aware of this per the discussion on this PR.